<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>memcreator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="kartinka">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>449</y>
      <width>400</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать картинку</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Тут должна быть ваша картинка&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bibl">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>490</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Библиотека</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>500</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ну или вы можете выбрать что-то из библиотеки!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>279</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;функционал&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sohranit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>420</y>
      <width>200</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить мем</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flag_ok">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>420</y>
      <width>100</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>Закончить</string>
    </property>
   </widget>
   <widget class="QSlider" name="povorot_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>260</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ugol_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Угол&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ugol">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>140</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Угол&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSlider" name="povorot">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>140</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="shrift">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Шрифт</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="menu">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>111</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>Меню</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vst_pic">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>330</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Вставить картинку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ramka">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>330</y>
      <width>330</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Сделать рамку с текстом</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vst_text">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>330</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Вставить текст</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
